emuu-client/src/components/CreatorsPage/Feeds.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "react";
+import "./Feeds.scss";
+import { Avatar } from "@mui/material";
+import YouTubeJSON from "../data/youtube-videos.json";
```

Comment:
why is this here? We don't need placeholder YouTube videos anymore after the first prototype.